lib/health_checks/mongoid_custom_client.rb

```diff
@@ -0,0 +1,13 @@
+module MongoidCustomClient
+  module_function
+
+  def create(client_name, hosts)
+    Mongoid::Config.clients[client_name] = {
+      hosts: hosts,
+      database: "#{client_name}_db",
```

Comment:
This is not the name of any existing database we have like `metrilometa`, `metrilodata`, etc.

Reply:
not sure what you mean, should it be like the name of any existing db? If yes, what for? In the end all that matters is the server address

Reply:
Yes, the DB name doesn't matter if you don't make requests to the DB. So why then do you specify it at all?

Reply:
so they are distinguishable enough from the rest. I can just leave it as an empty string, but what does an empty string say. In this way it will be something like database_meta_health_checks_db.

Reply:
Who is looking at those so that they have to be distinguishable?

Reply:
I have no the power to predict such things, but in case of debugging, it'll matter(I believe so).

Reply:
Even if you're debugging this will only confuse you, it's not an existing database and if you try to make a query it will fail. If you're debugging you'll simply look at `hosts` which is the bit that matters